python/plugins/processing/gui/AlgorithmDialog.py

```diff
@@ -156,6 +157,11 @@ def runAlgorithm(self):
                 self.feedback.reportError(self.algorithm().provider().warningMessage())
 
             self.setProgressText(QCoreApplication.translate('AlgorithmDialog', 'Processing algorithm…'))
```

Comment:
What do you think about dropping this line? It seems redundant now.

Reply:
I agree. I'll drop it.